<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenPropertiesDialog</class>
 <widget class="QDialog" name="PenPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>279</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>339</width>
    <height>279</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>画笔属性</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>41</y>
     <width>222</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="pen_width_label">
          <property name="text">
           <string>&amp;Pen Width:</string>
          </property>
          <property name="buddy">
           <cstring>pen_width_slider</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pen_width_slider">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSpinBox" name="pen_width_spin_box">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="pen_style_label">
        <property name="text">
         <string>Pen Style:</string>
        </property>
        <property name="buddy">
         <cstring>pen_style_combo_box</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pen_style_combo_box">
        <property name="currentText">
         <string>SolidLine</string>
        </property>
        <item>
         <property name="text">
          <string>SolidLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotLine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DashDotDotLine</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="cap_style_label">
        <property name="text">
         <string>Cap Style:</string>
        </property>
        <property name="buddy">
         <cstring>cap_style_combo_box</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cap_style_combo_box">
        <item>
         <property name="text">
          <string>SquareCap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FlatCap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RoundCap</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="join_style_label">
        <property name="text">
         <string>Join Style:</string>
        </property>
        <property name="buddy">
         <cstring>join_style_combo_box</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="join_style_combo_box">
        <item>
         <property name="text">
          <string>BevelJoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MiterJoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RoundJoin</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PenPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PenPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
